src/main/java/appeng/integration/BaseModule.java

```diff
@@ -21,12 +21,6 @@
 
 public abstract class BaseModule implements IIntegrationModule
 {
-
-	protected void testClassExistence( Class<?> clz )
-	{
-		clz.isInstance( this );
-	}
-
 	@Override
 	public abstract void init() throws Throwable;
```

Comment:
Any reason to keep them in a standalone module and not just put them into `IIntegrationModule`?